### ISSUE TITLE
Zip4j not working on android 21 JDK7

### ISSUE THREAD

**Issue by omkar-tenkale** [srikanth-lingala/zip4j#484]
```
    java.lang.NoSuchMethodError: No virtual method toPath()Ljava/nio/file/Path; in class Ljava/io/File; or its super classes (declaration of 'java.io.File' appears in /system/framework/core-libart.jar)
        at net.lingala.zip4j.util.FileUtils.getRelativeFileName(FileUtils.java:225)
        at net.lingala.zip4j.tasks.AbstractAddFileToZipTask.cloneAndAdjustZipParameters(AbstractAddFileToZipTask.java:218)
        at net.lingala.zip4j.tasks.AbstractAddFileToZipTask.addFilesToZip(AbstractAddFileToZipTask.java:71)
        at net.lingala.zip4j.tasks.AddFolderToZipTask.executeTask(AddFolderToZipTask.java:28)
        at net.lingala.zip4j.tasks.AddFolderToZipTask.executeTask(AddFolderToZipTask.java:17)
        at net.lingala.zip4j.tasks.AsyncZipTask.performTaskWithErrorHandling(AsyncZipTask.java:51)
        at net.lingala.zip4j.tasks.AsyncZipTask.execute(AsyncZipTask.java:45)
        at net.lingala.zip4j.ZipFile.addFolder(ZipFile.java:391)
        at net.lingala.zip4j.ZipFile.addFolder(ZipFile.java:366)
```

Is there any workaround for this? Desugaring is not working

**Comment by omkar-tenkale** [comment]
`java.nio.file` was added in android for in API 26

https://developer.android.com/reference/java/nio/file/package-summary.html

Any solution?

**Comment by srikanth-lingala** [comment]
Fixed in v2.11.4 released today

**Comment by omkar-tenkale** [comment]
thank you
